collector/src/main/java/com/bitmovin/analytics/data/EventData.java

```diff
@@ -37,6 +37,7 @@
     private String player;
     private String cdnProvider;
     private String videoId;
+    private String title;
```

Comment:
Since `EventData` has setters for all fields at the moment, we should consider adding also one for `title`.